src/components/organisms/Carousel.module.css

```diff
@@ -0,0 +1,81 @@
+.carouselContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.carouselWrapper {
+  display: flex;
+  width: 100%;
+  position: relative;
+  padding: 1rem;
+  background-color: var(--brand-white);
+}
+
+.carouselContentWrapper {
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+}
+
+.carouselContent {
+  display: flex;
+  transition: all 250ms linear;
```

Comment:
thought(blocking): lets stay with our usual syntax of `0.25s` here. Also may be worth to use a global `transition-duration` var in general